services/web/server/src/simcore_service_webserver/projects/projects_settings.py

```diff
@@ -8,3 +8,7 @@ class ProjectsSettings(BaseCustomSettings):
         description="defines the maximum authorized project data size"
         " when copying a project (disable with 0)",
     )
+    PROJECTS_MAX_NUM_RUNNING_DYNAMIC_NODES: NonNegativeInt = Field(
+        default=5,
+        description="defines the number of dynamic services in a project that can be started concurrently (a value of 0 will disable it)",
+    )
```

Comment:
May I suggest None to disable it? For some applications you might want to start 0 services.

Reply:
I am not sure I see which applications you are talking about? if I set it to 0 then you can effectively not start anything...